lesson9/main.py

```diff
@@ -0,0 +1,26 @@
+from database import Database
+from registration import RegistrationSystem
+from authentication import AuthenticationSystem
+
+# Default database -----------------------------------------------------------------------------------------------------
+
+# from user import User
+#
+# user1 = User("Alex", "111111", "26")
+# user2 = User("Max", "222222", "25")
+#
+# Database.users.append(user1.get_as_dict())
+# Database.users.append(user2.get_as_dict())
+#
+# Database.dump()
+
+# ---------------------------------------------------------------------------------------------------------------------
+
+Database.load()

```

Comment:
if our db doesnt exist - we just print message and continue work?.. better to stop it to create file. You can create file yourself into load method instead of just print message to console